server/services/participants/participant-entries.ts

```diff
@@ -188,13 +186,17 @@ export const getParticipants = async (
   isIndigenousFilter?: IsIndigenousFilter,
   programFilter?: ProgramFilter
 ) => {
+  // MHAW program isn't open yet to other regions
+  const hcaOnly =
+    user?.isHA && !MHAW_ENABLED_REGIONS.some((region) => user?.roles.includes(region));
```

Comment:
we should be checking for `isMHSUEmployer` too no?

Reply:
Why? I think the region has a higher priority than the employer type.

Reply:
I don't see anywhere limiting MHSU employer's by interior and costal.  If a MHSU employer from Fraser logs in, these changes won't show participants to them?